src/bidiMapper/domains/script/PreloadScriptStorage.ts

```diff
@@ -31,7 +31,7 @@ export class PreloadScriptStorage {
   readonly #scripts = new Set<PreloadScript>();
 
   /** Finds all entries that match the given filter. */
-  findPreloadScripts(filter?: BidiPreloadScriptFilter): PreloadScript[] {
+  find(filter?: BidiPreloadScriptFilter): PreloadScript[] {
```

Comment:
Used as PreloadScriptStorage.findPreloadScript double points to PreloadScript.

Reply:
LGTM, but:

`find` <-> `remove`

`findAll` <-> `removeAll`

It seems odd to have one of them with `All`, but not the other.

I feel `removeAll` -> `remove` would be a better change.

Reply:
Nice catch, I missed the `s` at the end and type. Agree `remove` would be better